test/mocha.opts

```diff
@@ -0,0 +1,5 @@
+--require ./node_modules/ts-node/register	
```

Comment:
You don't need to add this.

Reply:
Interesting, I didn't add this.  Any ideas why this was auto-generated?

Reply:
(removed anyway)